include/msckd.h

```diff
@@ -54,8 +54,10 @@
 #ifndef MSCKD_H_
 #define MSCKD_H_
 
+#include "opencv2/core/core.hpp"
```

Comment:
Sorry for the delayed response. I'm going through the changes now. Just a question here: is there an advantage to including only certain openCV modules rather than just opencv.hpp?